cypress/support/fragments/settings/remote-storage/configuration.js

```diff
@@ -0,0 +1,99 @@
+const successfulDeleteCalloutMessage = 'Remote storage configuration was successfully deleted.';
+const configurationPane = Pane({ title: 'Configurations' });
+
+function deleteRemoteStorage(name) {
+  cy.do([
+    MultiColumnListCell({ content: name }).click(),
+    Pane({ title: name }).find(Button('Actions')).click(),
+    Button({ id: 'clickable-delete-storage' }).click(),
+    Modal({ title: `Remove ${name}` }).find(Button('Delete')).click()
+  ]);
+  InteractorsTools.checkCalloutMessage(successfulDeleteCalloutMessage);
+  cy.expect(configurationPane.find(MultiColumnListCell({ content: name })).absent());
+}
+
+const providers = {
+  DematicEMS: {
+    title: 'Dematic EMS',
+    create(name) {
+      cy.do([
+        Button('+ New').click(),
+        TextField({ name: 'name' }).fillIn(name),
+        Accordion({ label: 'General information' }).find(Select()).choose('Dematic EMS'),
+        Button('Save & close').click(),
+        Button('Save').click()
+      ]);
+      InteractorsTools.checkCalloutMessage(successfulCreateCalloutMessage);
+    },
+    delete(name) {
+      deleteRemoteStorage(name);
+    }
+  },
+  DematicStagingDirector: {
+    title: 'Dematic StagingDirector',
+    create(name) {
+      cy.do([
+        Button('+ New').click(),
+        TextField({ name: 'name' }).fillIn(name),
+        Accordion({ label: 'General information' }).find(Select()).choose('Dematic StagingDirector'),
+        TextField({ name: 'accessionDelay' }).fillIn('1'),
+        Button('Save & close').click(),
+        Button('Save').click()
+      ]);
+      InteractorsTools.checkCalloutMessage(successfulCreateCalloutMessage);
+    },
+    delete(name) {
+      deleteRemoteStorage(name);
+    }
+  },
+  CaiaSoft: {
+    title: 'CaiaSoft',
+    create(name) {
+      cy.do([
+        Button('+ New').click(),
+        TextField({ name: 'name' }).fillIn(name),
+        Accordion({ label: 'General information' }).find(Select()).choose('CaiaSoft'),
+        Accordion({ label: 'Accession holding workflow preference' }).find(Select()).choose('Change permanent location'),
+        Accordion({ label: 'Returning workflow preference' }).find(Select()).choose('Items received at remote storage scanned into FOLIO'),
+        Button('Save & close').click(),
+        Button('Save').click()
+      ]);
+      InteractorsTools.checkCalloutMessage(successfulCreateCalloutMessage);
+    },
+    delete(name) {
+      deleteRemoteStorage(name);
+    }
+  }
+};
+
+export default {
+  providers,
+
+  verifyCreatedRemoteStorage(name, provider) {
+    const date = DateTools.getFormattedDate({ date: new Date() }, 'MM/DD/YYYY');
+
+    cy.expect(configurationPane.find(MultiColumnListCell({ content: name })).exists());
+    cy.expect(configurationPane.find(MultiColumnListCell({ content: provider.title })).exists());
+    cy.expect(configurationPane.find(MultiColumnListCell({ content: date })).exists());
+  },
+
+  tryToEditRemoteStorage(name) {
```

Comment:
it's more clear and simple to have set of methods instead of this special method. 

- select
- edit(may be method to fill common fields can be applicable in this case)
- closeWithoutSaving